packages/internal/package.json

```diff
@@ -14,10 +14,8 @@
   "dependencies": {
     "@babel/parser": "7.15.3",
     "@babel/plugin-transform-typescript": "7.15.0",
-    "babel-plugin-polyfill-corejs3": "0.2.4",
```

Comment:
Didn't I just add these packages?

Reply:
They're duped for some reason!

Reply:
(see line 29 and 30)

Reply:
Oh. The brain only parses the red and green lines.